tests/conftest.py

```diff
@@ -128,3 +125,49 @@ def synapse_store(request):
         synapse_store = SynapseStorage()
 
     yield synapse_store
+
+
+# These fixtures make copies of existing test manifests.
+# These copies can the be altered by a given test, and the copy will eb destroyed at the 
+# end of the test
+
+@pytest.fixture(scope="function")
+def test_bulkrnaseq(helpers: Helpers) -> Generator[str, None, None]:
```

Comment:
Hi @andrewelamb! I think this function can be re-written in a better way that not only works for "mock_manifests/test_BulkRNAseq.csv", but also works for other file path that we might want to use in the future: 
```
@pytest.fixture(scope="function")
def temporary_file_copy(request, helpers: Helpers) -> Generator[str, None, None]:
    file_name = request.param
    # original file copy
    original_test_path = helpers.get_data_path(f"mock_manifests/{file_name}")
    # Copy the original CSV file to a temporary directory
    temp_csv_path = helpers.get_data_path(f"mock_manifests/test_{file_name}_copy.csv")
    
    shutil.copyfile(original_test_path, temp_csv_path)
    yield temp_csv_path
    # Teardown
    if os.path.exists(temp_csv_path):
        os.remove(temp_csv_path)
```

I have tested that the above would work. To call the fixture in other function, you could do something like: 
```
    @pytest.mark.parametrize("temporary_file_copy", ["mock_manifests/test_BulkRNAseq.csv"])
    def test_submit_metadata_manifest(
        self,
        temporary_file_copy,
        helpers: Helpers,
        file_annotations_upload: bool,
        restrict_rules: bool,
        data_model_labels: str,
        hide_blanks: bool,
        validate_component: Optional[str],
    ) -> None:
```

Reply:
What is the request here `def temporary_file_copy(request, helpers: Helpers) -> Generator[str, None, None]:` ?

Reply:
Also that sounds good, but I think we're beyond the scope of this issue. Since this PR is almost a month old,  think we should get this approved and save the above improvment for another issue. Thanks!

Reply:
More about request: https://docs.pytest.org/en/6.2.x/example/simple.html#request-example